packages/application/src/shell.ts

```diff
@@ -770,6 +774,31 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     }
   }
 
+  /**
+   * Activate Specified Area Widget.
+   */
+  activateWidget(area?: ILabShell.Area): void {
```

Comment:
Naming suggestion: `activateArea`?

Reply:
Actually since this is an API change quite close to the core, I will wait with setting milestone here. It may be better to put it in 4.1

Reply:
Changed to activateArea